src/arrays.js

```diff
@@ -1,59 +1,101 @@
+
+
 export const getNthElement = (index, array) => {
-  // your code here
+  if (index + 1 > array.length){
+    return array[index - array.length];
+  }
+  return array[index];
 };
 
 export const arrayToCSVString = (array) => {
-  // your code here
+  return array.toString('');
 };
 
 export const csvStringToArray = (string) => {
-  // your code here
+  string = string.replace(/,/g, '');
+  
+  return string.split('');
 };
 
 export const addToArray = (array, element) => {
-  // your code here
+  element.push(array);
 };
 
-export const addToArray2 = (array, element) => {
-  // your code here
+export const addToArray2 = (element, array) => {
+  let arrayClone = array.slice();
+  arrayClone.push(element);
+  return arrayClone;
 };
 
 export const removeNthElement = (index, array) => {
-  // your code here
+   
+  return array.splice(index, 1);
 };
 
 export const numbersToStrings = (numbers) => {
-  // your code here
+  return numbers.toString().split(',');
 };
 
 export const uppercaseWordsInArray = (strings) => {
-  // your code here
+  let stringsUpper = String(strings).toUpperCase().split(",");
```

Comment:
You could use a constant here, and there's no need to assign to a variable - you could just `return String(strings).toUpperCase().split(",")`